controllers/front/ProductController.php

```diff
@@ -954,10 +954,15 @@ private function getIdProductAttribute($useGroups = false)
             $groups = Tools::getValue('group');
 
             if (!empty($groups)) {
-                $requestedIdProductAttribute = (int) Product::getIdProductAttributesByIdAttributes(
-                    $this->product->id,
-                    $groups
-                );
+                try {
+                    $requestedIdProductAttribute = (int) Product::getIdProductAttributeByIdAttributes(
+                        $this->product->id,
+                        $groups
+                    );
+                } catch (PrestaShopObjectNotFoundException $e) {
+                    // Don't care just set $requestedIdProductAttribute to null
```

Comment:
Shouldn't be logged at least? I'm not easy to hide an exception.

Reply:
There was no log in 1.7.3 and don't found anything in legacy to add a log without changing many things (the only log function is protected under PrestasShopException)

Reply:
Hey, Thanks mate ;)